apps/www/middleware.ts

```diff
@@ -133,13 +133,20 @@ async function middlewareFunc(req: NextRequest): Promise<NextResponse> {
   }
 
   const resUrl = req.nextUrl.clone()
+
   // Block if request is coming from IP_BLOCKLIST
   const clientIp = req.headers['x-forwarded-for']
     ? req.headers['x-forwarded-for'].split(',')[0]
     : ''
 
-  if (clientIp === '129.80.232.132') {
-    console.log(`request with IP: ${clientIp} denied.`)
+  const isBlacklistedIP =
+    clientIp &&
+    process.env.IP_BLOCKLIST &&
+    (process.env.IP_BLOCKLIST || '')
+      .split(',')
```

Comment:
i am not sure about the splitting. i feel like this also adds potential for erros if someone adds a whitespace after the comma or something. 
i think i would do:
`process.env.IP_BLOCKLIST.includes(clientIp)` without the splitting. Since IPs don't include commas this should work too, no?

Reply:
i see what you mean. makes sense